src/sinks/aws_kinesis/firehose/integration_tests.rs

```diff
@@ -189,12 +198,12 @@ async fn ensure_elasticsearch_domain(domain_name: String) -> String {
                 .map(|v| {
                     v.get("status")
                         .and_then(|status| status.as_str())
-                        .map(|status| status == "green")
+                        .map(|status| status == "green" || status == "yellow")
```

Comment:
in a recent external contributor PR we used 🤷 

```suggestion
                        .map(|status| status != "red")
```